krossbow-stomp-kxserialization/build.gradle.kts

```diff
@@ -12,6 +12,7 @@ kotlin {
         browser()
     }
     ios()
+    iosSimulatorArm64()
```

Comment:
This is surprising. I should have referenced `setupNativeTargets()` there instead 🤔 

Did you by any chance try to do that already? If not, it's ok. I'll merge as-is and make the change myself before releasing

Reply:
I was wondering about that too! I just assumed there was a reason for it, but I can definitely try to use `setupNativeTargets()` here!

Reply:
I was merely asking you in case you had already tried, but if you haven't that's ok 😉